tests/cypress/fixtures/si.json

```diff
@@ -0,0 +1,54 @@
+{
+    "lattice": {
+        "a": 3.867,
+        "b": 3.867,
+        "c": 3.867,
+        "alpha": 60,
+        "beta": 60,
+        "gamma": 60,
+        "units": {
+            "length": "angstrom",
+            "angle": "degree"
+        },
+        "type": "FCC",
+        "vectors": {
+            "a": [3.34892, 0, 1.9335],
+            "b": [1.116307, 3.157392, 1.9335],
+            "c": [0, 0, 3.867],
+            "alat": 1,
+            "units": "angstrom"
+        }
+    },
+    "basis": {
+        "elements": [
+            {
+                "id": 0,
+                "value": "Si"
+            },
+            {
+                "id": 1,
+                "value": "Si"
+            }
+        ],
+        "coordinates": [
+            {
+                "id": 0,
+                "value": [0, 0, 0]
+            },
+            {
+                "id": 1,
+                "value": [0.25, 0.25, 0.25]
+            }
+        ],
+        "units": "crystal",
+        "cell": [
+            [3.348920236434424, 0, 1.9335000000000004],
+            [1.1163067454781415, 3.1573922784475164, 1.9335000000000004],
+            [0, 0, 3.867]
+        ],
+        "constraints": []
+    },
+    "name": "Silicon FCC",
+    "_id": "",
+    "metadata": {}
+}
```

Comment:
The screenshots below need to be cleaned up together with fixtures. Let's also clean git history pls